src/AssetsPage/AssetsTable/AssetsTable.test.jsx

```diff
@@ -2,8 +2,11 @@ import React from 'react';
 import { mount } from 'enzyme';
 import { AssetsTable } from './index';
 
-let wrapper;
+import * as AssetsConstants from '../../data/constants/assets'; 
 
+const setAssetToDelete = (asset, wrapper) => {
```

Comment:
I figured we may want to test this in other tests moving forward so I moved it out to reduce the duplication that I was already doing - it could easily go back into each test, I wasn't sure which was preferable for testing.  I could see it making sense to have it in each since they're supposed to be individual tests, but this seemed cleaner for the duplication.